CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 ### Added
 - Add a way to retrieve a defined task [#1008](https://github.com/deployphp/deployer/pull/1008)
 - Add support for configFile in the NativeSsh implementation [#979](https://github.com/deployphp/deployer/pull/979)
+- Add `--no-hooks` option for running commands without `before()` and `after()` [#934](https://github.com/deployphp/deployer/pull/934)
```

Comment:
@elfet I don't understand this rule. I think that the link must point to an issue or PR which describes the problem. Therefore I pointed out previous PR. Isn't it correct?